openpype/hosts/nuke/plugins/publish/extract_review_data.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pyblish.api
+import openpype
+from pprint import pformat
+
+
+class ExtractReviewData(openpype.api.Extractor):
+    """Extracts review tag into available representation
+    """
+
+    order = pyblish.api.ExtractorOrder + 0.01
+    # order = pyblish.api.CollectorOrder + 0.499
+    label = "Extract Review Data"
+
+    families = ["review"]
+    hosts = ["nuke"]
+
+    def process(self, instance):
+        fpath = instance.data["path"]
+        ext = os.path.splitext(fpath)[-1][1:]
+
+        representations = instance.data.get("representations", [])
+
+        if "render.farm" in instance.data["families"]:
```

Comment:
Won't this cause that farm publishing will not produce review?

Reply:
this is done by `ProcessSubmittedJobOnFarm` so no need to have `review` in families. If you keep it there it will be switching `ExtractReview` during publishing on local processing, which is not desirable.

Reply:
Then maybe add comment here which explaint why it's removed and also would be good to check if `"review"` is in families before removing it?

Reply:
good point, thanks ;)